modules/pricer/src/main/java/com/opengamma/strata/pricer/swaption/VolatilitySwaptionPhysicalProductPricer.java

```diff
@@ -144,6 +144,18 @@ public double impliedVolatility(
     return swaptionVolatilities.volatility(expiry, tenor, strike, forward);
   }
 
+  //-------------------------------------------------------------------------
+  /**
+   * Provides the forward rate of the next fixing date
+   *
+   * @param swaption  the swaption
+   * @param ratesProvider  the rates provider
+   * @return the forward rate
+   */
+  public double forwardRate(ResolvedSwaption swaption, RatesProvider ratesProvider) {
+    return VolatilitySwaptionProductPricer.DEFAULT.forwardRate(swaption, ratesProvider);
```

Comment:
Instead of using `VolatilitySwaptionProductPricer.DEFAULT`, there should be a local variable of type `VolatilitySwaptionProductPricer`, as per `DiscountingSwapProductPricer`